Description: Comment out references to fonts.googleapis.com to prevent lintian
 from warning about potential privacy-breach-generic issues.
Author: Hugh McMaster
Bug-Debian: https://bugs.debian.org/873432
Forwarded: not-needed
Last-Update: 2018-07-02

--- a/ft2docs/docs/css/freetype2.css
+++ b/ft2docs/docs/css/freetype2.css
@@ -5,9 +5,9 @@
 
 
 /* use Droid fonts */
-@import url("https://fonts.googleapis.com/css?family=Droid+Serif:r,i,b,bi");
-@import url("https://fonts.googleapis.com/css?family=Droid+Sans:r,b");
-@import url("https://fonts.googleapis.com/css?family=Droid+Sans+Mono");
+/*@import url("https://fonts.googleapis.com/css?family=Droid+Serif:r,i,b,bi");*/
+/*@import url("https://fonts.googleapis.com/css?family=Droid+Sans:r,b");*/
+/*@import url("https://fonts.googleapis.com/css?family=Droid+Sans+Mono");*/
 
 
 /* top-level appearance */
